tests/test_vpn_model.py

```diff
@@ -10,13 +10,6 @@
 
 def gen_mock_values(values):
     """Generator to return the next value in a list of values on every call.
-
-    >>> vals = gen_mock_values([1, 2, 3])
-    >>> mocked_func.side_effect = lambda: next(vals)
-    >>> mocked_func()
-        1
-    >>> mocked_func()
-        2
```

Comment:
I actually did this because PyCharm was actually running those comments while running tests and kept raising exceptions; I have no problem keeping them there otherwise.